### PR TITLE
HDDS-1853. Fix failing blockade test-cases.

### PR DESCRIPTION
**Pull request by nandakumar131**
This PR is to fix and make sure that all the test-cases in blockade are working.

### CODE REVIEW

File: hadoop-ozone/dist/src/main/compose/ozoneblockade/docker-config [hunk 1]
```diff
@@ -32,9 +32,9 @@ OZONE-SITE.XML_ozone.scm.pipeline.owner.container.count=1
 OZONE-SITE.XML_ozone.scm.pipeline.destroy.timeout=15s
 OZONE-SITE.XML_hdds.heartbeat.interval=2s
 OZONE-SITE.XML_hdds.scm.wait.time.after.safemode.exit=30s
-OZONE-SITE.XML_hdds.scm.replication.thread.interval=5s
-OZONE-SITE.XML_hdds.scm.replication.event.timeout=7s
-OZONE-SITE.XML_dfs.ratis.server.failure.duration=25s
+OZONE-SITE.XML_hdds.scm.replication.thread.interval=6s
```
Comment (mukul1987) [inline hadoop-ozone/dist/src/main/compose/ozoneblockade/docker-config]: Can you please explain the need of increasing these values ?

Comment (nandakumar131) [reply hadoop-ozone/dist/src/main/compose/ozoneblockade/docker-config]: In some test-cases we want to write to a pipeline where there are only two nodes in the ring (we cut the connection to the third node). In such cases SCM was destroying the pipeline immediately and the replication manager kicks-in. These values are adjusted such that the client gets enough time to write to a pipeline where there are only two healthy nodes and verify the writes.

**Comment by nandakumar131** [comment]
Test failures are not related.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 41 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 1 | No case conflicting files found. |
| 0 | shelldocs | 0 | Shelldocs was not available. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 8 new or modified test files. |
||| _ trunk Compile Tests _ |
| 0 | mvndep | 38 | Maven dependency ordering for branch |
| +1 | mvninstall | 594 | trunk passed |
| +1 | compile | 371 | trunk passed |
| +1 | mvnsite | 0 | trunk passed |
| -1 | pylint | 4 | Error running pylint. Please check pylint stderr files. |
| +1 | shadedclient | 743 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 169 | trunk passed |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 25 | Maven dependency ordering for patch |
| +1 | mvninstall | 552 | the patch passed |
| +1 | compile | 377 | the patch passed |
| +1 | javac | 377 | the patch passed |
| +1 | mvnsite | 0 | the patch passed |
| -1 | pylint | 9 | Error running pylint. Please check pylint stderr files. |
| +1 | pylint | 9 | There were no new pylint issues. |
| +1 | shellcheck | 0 | There were no new shellcheck issues. |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 638 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 160 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 284 | hadoop-hdds in the patch passed. |
| -1 | unit | 3536 | hadoop-ozone in the patch failed. |
| +1 | asflicense | 46 | The patch does not generate ASF License warnings. |
| | | 7811 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.ozone.container.ozoneimpl.TestOzoneContainer |
|   | hadoop.ozone.container.common.statemachine.commandhandler.TestBlockDeletion |
|   | hadoop.ozone.om.TestOzoneManager |
|   | hadoop.ozone.client.rpc.TestCloseContainerHandlingByClient |
|   | hadoop.hdds.scm.pipeline.TestNodeFailure |
|   | hadoop.ozone.client.rpc.TestOzoneClientRetriesOnException |
|   | hadoop.ozone.client.rpc.TestOzoneRpcClient |
|   | hadoop.ozone.client.rpc.TestMultiBlockWritesWithDnFailures |
|   | hadoop.ozone.TestStorageContainerManager |
|   | hadoop.ozone.client.rpc.TestOzoneRpcClientWithRatis |
|   | hadoop.hdds.scm.pipeline.TestRatisPipelineCreateAndDestory |
|   | hadoop.ozone.client.rpc.TestBCSID |
|   | hadoop.ozone.container.common.statemachine.commandhandler.TestCloseContainerHandler |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.0 Server=19.03.0 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1151/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1151 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient shellcheck shelldocs pylint |
| uname | Linux e011f9310ef7 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / cf9ff08 |
| Default Java | 1.8.0_212 |
| pylint | https://builds.apache.org/job/hadoop-multibranch/job/PR-1151/1/artifact/out/branch-pylint-stderr.txt |
| pylint | https://builds.apache.org/job/hadoop-multibranch/job/PR-1151/1/artifact/out/patch-pylint-stderr.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1151/1/artifact/out/patch-unit-hadoop-ozone.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1151/1/testReport/ |
| Max. process+thread count | 3951 (vs. ulimit of 5500) |
| modules | C: hadoop-ozone/fault-injection-test/network-tests hadoop-ozone/dist U: hadoop-ozone |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1151/1/console |
| versions | git=2.7.4 maven=3.3.9 shellcheck=0.4.6 pylint=1.9.2 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Pull request merged**